Makefile

```diff
@@ -74,7 +74,11 @@ help:  ## Display this help
 
 .PHONY: test
 test: generate lint ## Run tests
-	go test -v ./...
+	source ./scripts/fetch_ext_bins.sh && \
+	fetch_tools && \
```

Comment:
Seems like if you are running `make test` every time to test, the tools are downloaded regardless of its existence, which can be annoying. Is it possible to fix the script to skip downloading of tools?

Reply:
Absolutely! I just didn't want to be too invasive. Happy to make that change.

Reply:
Alright, the changes are a bit more invasive now, but they are better from your suggestion. Thank you!